[docs][location] clarify app termination (#34436)
expo/expo: docs/pages/versions/unversioned/sdk/location.mdx

</ConfigReactNative>

### Background location

Background location allows your app to receive location updates while it is running in the background and includes both location updates and region monitoring through geofencing. This feature is subject to platform API limitations and system constraints:

- Background location will stop if the user terminates the app.

> - Background location resume if the user restarts the app.

Reviewer: ```suggestion
- Background location resumes if the user restarts the app.
```

Grammatical error.

- On Android, a terminated app will not automatically restart when a location or geofencing event occurs due to platform limitations. <PlatformTag platform="android" />

> - On iOS, terminated apps will be restarted by the system when new geofence event occurs. <PlatformTag platform="ios" />

Reviewer: ```suggestion
- On iOS, the system will restart terminated apps when a new geofence event occurs. <PlatformTag platform="ios" />
```

Missing article and we can also avoid passive voice usage in the first half of the sentence.

> - On iOS, terminated apps will be restarted by the system when new geofence event occurs. <PlatformTag platform="ios" />

Reviewer: I don't think adding Platform tags in the sentences has significance when the sentence is already starting with and mentioning the correct platform (Eg, On Android...).

I think we can remove the platform tags from the from the end of these sentences or move them in front of the sentences and remove On Android and on iOS. Another reason is that an inconsistent pattern has started to emerge. For example, take a look at the screenshot below in which the section (http://docs.expo.dev/versions/unversioned/sdk/location/#background-location-methods) has platform tags in from of them

![CleanShot 2025-01-24 at 23 28 14@2x](https://github.com/user-attachments/assets/b7855b87-5505-4478-9202-f13e6c94f43a)

Author: Agree - I think I like having them in the front - it gives more clarity because of the symmetry in layout.
